src/pages/Animal/Feed/index.tsx

```diff
@@ -22,8 +24,7 @@ const FeedPets = (): JSX.Element => {
   }, [navigation]);
 
   const fetchPets = (): void => {
-    firestore().collection('animals').orderBy('name')
-      .get()
+    animalAPI.animalFirestoreCollection().orderBy('name').get()
```

Comment:
I suggest `animalAPI.all()`, or even `Animals.all()` and by default ordered by name

Reply:
Fixed by new commit.